pkg/awsutils/awsutils.go

```diff
@@ -124,6 +124,8 @@ type APIs interface {
 
 	// GetENILimit returns the number of enis can be attached to an instance
 	GetENILimit() (int, error)
+	// GetPrimaryENIMAC returns the mac address of primary eni
```

Comment:
Nit: `// GetPrimaryENIMAC returns the mac address of the primary ENI`

Reply:
done